dpytools/logging/logger.py

```diff
@@ -86,17 +115,17 @@ def _log(
         if self.flush_stdout_after_log_entry is True:
             sys.stdout.flush()
 
-    def debug(self, event: str, raw: str = None, data: Dict = None):
+    def debug(self, event: str, raw: str = None, data: Dict = None, response: requests.Response = None):
```

Comment:
In the log_event dictionary above, the r_dict object in line 108 is assigned to the key `response_dict`, but in these logging functions (debug, info, warning) the response details are written simply as `response`, could that potentially lead to any issues or discrepancies? Not saying it should change, just wondering if anything could happen as a result of the argument assignment not having the same exact names. if not then safely ignore this

Reply:
I don't think it would lead to any issue but yeah if it is a concern I can change it.